examples/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:8
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY $directory $directory
+RUN npm install
+EXPOSE 4000
+CMD [ "node", "$directory" ]
```

Comment:
this is not a valid dockerfile, it probably better rename it to be something like `Dockerfile.template`. Otherwise people may try to run `docker build` from this dir

Reply:
using args now